ingest/api/std_name_units.json

```
@@ -0,0 +1,481 @@
+        "symbol": "cm"
+    },
+    "surface_temperature": {
+        "unit": "degC",
+        "symbol": "degC",
+        "conversion": {
+            "K": {
+                "add": -273.15
+            }
+        }
+    },
+    "surface_upwelling_longwave_flux_in_air": {
+        "unit": "W/m2",
+        "alias": [
+            "W m-2",
+            "Wm-2"
+        ],
+        "symbol": "W/m2"
+    },
+    "surface_upwelling_radiance_in_air": {
+        "unit": "kJ/m2",
+        "alias": [
+            "kJm-2",
+            "kJ m-2"
+        ],
+        "symbol": "kJ/m2"
+    },
+    "surface_upwelling_shortwave_flux_in_air": {
+        "unit": "W/m2",
+        "alias": [
+            "W m-2",
+            "Wm-2"
+        ],
+        "symbol": "W/m2"
+    },
+    "tendency_of_air_pressure": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "alias": [],
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "tendency_of_surface_air_pressure": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "thickness_of_snowfall_amount": {
+        "unit": "cm",
+        "symbol": "cm"
+    },
+    "ultraviolet_index": {
+        "unit": "index",
+        "symbol": "index"
+    },
+    "virtual_temperature": {
+        "unit": "degC",
+        "symbol": "degC",
+        "conversion": {
+            "K": {
+                "add": -273.15
+            }
+        }
+    },
+    "visibility_in_air": {
+        "unit": "m",
+        "symbol": "m"
+    },
+    "wet_bulb_temperature": {
+        "unit": "degC",
+        "symbol": "degC",
+        "conversion": {
+            "K": {
+                "add": -273.15
+            }
+        }
+    },
+    "wind_from_direction": {
+        "unit": "degrees",
+        "symbol": "degrees",
+        "alias": [
+            "deg",
+            "degree"
+        ]
+    },
+    "wind_speed": {
+        "unit": "m/s",
+        "alias": [
+            "m s-1",
+            "ms-1"
+        ],
+        "symbol": "m/s"
+    },
+    "wind_speed_of_gust": {
+        "unit": "m/s",
+        "alias": [
+            "m s-1",
+            "ms-1"
+        ],
+        "symbol": "m/s"
+    },
+    "air_pressure": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "alias": [],
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "air_pressure_at_mean_sea_level": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "alias": [],
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "air_temperature": {
+        "unit": "degC",
+        "conversion": {
+            "K": {
+                "add": -273.15
+            }
+        }
+    },
+    "cloud_area_fraction": {
+        "unit": "octas",
+        "alias": "octa"
+    },
```

```suggestion
    "cloud_area_fraction": {
        "unit": "oktas",
        "symbol": "oktas",
        "alias": [
            "okta",
            "octa",
            "octas"
        ]
    },
```

https://en.wikipedia.org/wiki/Okta